application/classes/Ushahidi/Repository/Tag.php

```diff
@@ -162,8 +160,30 @@ public function delete(Entity $entity)
 	public function deleteTag($id)
 	{
 		// Remove tag from attribute options
-		$this->removeTagFromAttributeOptions($entity->id);
-
+		$this->removeTagFromAttributeOptions($id);
```

Comment:
I'm really not sure why this was using $entity->id since there was not entity to use. I could revert it .. it's probably not being used . :/

Reply:
Hmmm yeah this must not have been used anymore